packages/builder-vite/vite-config.ts

```diff
@@ -68,6 +68,15 @@ export async function pluginConfig(options: ExtendedOptions, _type: PluginConfig
       // Do not treat story files as HMR boundaries, storybook itself needs to handle them.
       exclude: [/\.stories\.([tj])sx?$/, /node_modules/].concat(framework === 'react' ? [] : [/\.([tj])sx?$/]),
     }),
+    {
+      name: 'vite-plugin-storybook-allow',
+      enforce: 'post',
+      config(config) {
+        if (config?.server?.fs?.allow) {
```

Comment:
Ok, so this will catch situations like the sveltekit plugin adding an allow list, which then prevents the root from being used.  I like this solution, but would you mind adding a quick comment somewhere here that explains why it's needed?  Otherwise, there's only about a 10% chance I'll remember why we're doing this in a few months.  :).

Reply:
Nevermind, I'll add the comment.  I want to get a release out with the MDX fix, and would be nice to include this as well.

Reply:
done

Reply:
Hah, you beat me to it!  Thanks.